examples/font_viewer.py

```diff
@@ -241,11 +244,12 @@ def handle_events(self):
             elif e.type == pg.KEYDOWN:
                 if e.key == pg.K_ESCAPE:
                     return False
-            elif e.type == pg.MOUSEBUTTONDOWN:
-                # enter or exit virtual mouse mode for scrolling
+            elif e.type == pg.MOUSEWHEEL:
+                self.y_offset += e.y * self.MOUSE_SCROLL_SPEED * -1
+            elif e.type == pg.MOUSEBUTTONDOWN or e.type == pg.MOUSEBUTTONUP:
+                # enter or exit virtual mouse mode for click and drag
```

Comment:
It's a minor thing, but I'd probably handle up and own separately. Since this is an example script, I think it's a good idea to set a good example.

Reply:
Ah I see, that makes sense! I separated them out, hopefully that is what you were thinking of.